examples/galaxies/plot_ellipticity.py

```diff
@@ -0,0 +1,128 @@
+"""
+Galaxy Ellipticity Distributions
+================================
+
+This example demonstrate how to sample ellipticity distributions
+in SkyPy.
+
+"""
+
+
+# %%
+# 3D Ellipticity Distribution
+# ---------------------------
+#
+# In Ryden 2004 [1]_, the ellipticity is sampled by randomly projecting
+# a 3D ellipsoid with principal axes :math:`A > B > C`.
+#
+# The distribution of the axis ratio :math:`\gamma = C/A` is a truncated
+# normal with mean :math:`\mu_\gamma` and standard deviation
+# :math:`\sigma_\gamma`.
+#
+# The distribution of :math:`\epsilon = \log(1 - B/A)` is truncated normal
+# with mean :math:`\mu` and standard deviation :math:`\sigma`.
+#
+#
+# The func:`skypy.galaxies.morphology.ryden04_ellipticity()` model samples
+# projected 2D axis ratios. Specifically, it samples the axis ratios of the
+# 3D ellipsoid according to the description above [1] and
+# then randomly projects them using
+# :func:`skypy.utils.random.triaxial_axis_ratio()`.
```

Comment:
Potentially confusing to mention the `triaxial_axis_ratio` function. It is hidden from the user who doesn't need to understand the implementation in this context.

Reply:
Not sure I agree, but I'm open to delete that bit.

Reply:
To be clear, we still want to explain the model (population of triaxial ellipsoids with some distribution in projection) but the implementation details (`ryden04_ellipticity` calls another function `triaxial_axis_ratio`) are at best irrelevant and at worst confusing for the user.